src/components/Modals/FiatRamps/views/Overview.tsx

```diff
@@ -271,6 +272,14 @@ export const Overview: React.FC<OverviewProps> = ({
     [asset?.symbol, fiatRampAction, translate],
   )
 
+  const description: string | [string, InterpolationOptions] | undefined = useMemo(() => {
+    if (!asset) return
+    if (!isConnected) return
+    if (isUnsupportedAsset)
+      return ['fiatRamps.notSupported', { asset: asset.symbol, wallet: wallet?.getVendor() }]
+    return fundsTranslation
+  }, [asset, fundsTranslation, isConnected, isUnsupportedAsset, wallet])
+
```

Comment:
Nada for unsupported wallet, the connect button is self explanatory

<img width="433" alt="image" src="https://github.com/user-attachments/assets/22a0942d-e18c-4166-843e-b8f321596f13">
<img width="443" alt="image" src="https://github.com/user-attachments/assets/6691f21a-51d0-41d4-8431-4d579262867a">